CMakeLists.txt

```diff
@@ -1088,6 +1090,74 @@ elseif(SunOS)
         src/util/platform/FFPlatform_unix.c
         src/util/binary_linux.c
     )
+elseif(Haiku)
```

Comment:
Does Haiku support OpenGL, pulseaudio or something?

Reply:
OpenGL yes, we have our own audio framework (MediaKit)… I wanted to have something that compiles before expanding to these.